Courir/Courir/MenuViewController.swift

```diff
@@ -10,13 +10,21 @@ import UIKit
 import SpriteKit
 
 class MenuViewController: UIViewController {
-
+    private var nameTextField: UITextField?
```

Comment:
Removed text field as instance variable. Save action needs to stay as instance variable for selector function `textFieldDidChange:` to work